File: js/dead.js
function deadinside(num) {
  if(num < 7) return;
  let num2 = num - 7;
  console.log(num + ' - 7 = ' + num2);
  deadinside(num2);
}

console.log('Я гуль');
deadinside(1000);

File: cpp/dead.cpp
#include <cstdio>
using namespace std;
void kuraysdead(int num) {
 if(num<7) return;
 int num2 = num - 7;
 printf("%d %s %d %s",num," - 7 = ",num2,"\n");
 kuraysdead(num2);
}
int main() {
 printf("Я гуль!\n");
 kuraysdead(1000);
 return 0;
}
